### ISSUE TITLE
[Markup and meta data] Name and role can't be programatically determined for UI components. (09.06.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#34162]
#### VFS team name
Debt Resolution

#### VFS product name
Medical Copays (MCP)

#### Point of Contact/Reviewers
Josh Kim (@joshkimux) - Accessibility

---

### Platform Issue
Name and role can't be programatically determined for UI components.

### Issue Details
The `p` element's id `copay-balance` is not unique for each of the cards. This will result in all of the headings being described by only the first `p` element.

### VA.gov Experience Standard
[Category Number 09, Issue Number 06](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 4.1.2_A

---

### Platform Recommendation
All `id`s must be unique. Multiple `p` elements that contain unique facility names should not share the same `id`.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by craigwheeler** [comment]
hey @joshkimux or @AngelaFowler82  just looking into this and not sure this issue still exists since i'm seeing that `copay-balance` does have a unique id for each of the cards so may have already been addressed? can one of you verify if this is still an issue thanks!

![Medical Copays  Veterans Affairs 2022-01-07 12-14-54(1).png](https://images.zenhubusercontent.com/5f355b2a6fe9a1b76ba980d4/af11efbc-2fe0-42f9-92cf-98f037432a26)

**Comment by joshkimux** [comment]
@craigwheeler yep! That should do the trick, but I'd recommending testing with @AngelaFowler82 first on a screen reader to confirm that the string "copay balance name" is unique for each receipt heading.

**Comment by AngelaFowler82** [comment]
User+81 appears to owe two debts to the same facility, though I can't tell if this is a result of this issue or simply an oversight from when the account was created.
![image.png](https://images.zenhubusercontent.com/60367a69c19840b3339a6a5e/df841982-d181-4de0-92a9-ea6da18edf04)

**Comment by craigwheeler** [comment]
Correct @AngelaFowler82 there is a zero balance included in the mock data for user+81 which results in 2 copays showing for the same facility but should be ignored since zero balances should not show on prod... @ScottyRJames can you confirm the mock user data will be updated to remove zero balances or can you advise zero balances should not show on prod? Thanks!

**Comment by ScottyRJames** [comment]
This zero balance is being brought back from the VBS endpoint. Have reached out to the Liberty team to remove this balance and only keep outstanding balances for this test user's payload.

**Comment by ScottyRJames** [comment]
VBS is not returning $0 balances from their endpoint and we have a feature flipper in place to avoid including them from our vets API endpoint so they will not be shown in production. This is just some leftover mocked data.

**Comment by AngelaFowler82** [comment]
In that case, I can officially close this baby!
